README.md

```diff
@@ -1,91 +1,154 @@
+
+## License
+
+This repository is licensed under the [Apache 2.0 License](https://github.com/mattermost/mattermost-plugin-bitbucket/blob/master/LICENSE).
+
+## About the Bitbucket Plugin
+
+The Mattermost Bitbucket plugin uses a webhook to connect your Bitbucket account to Mattermost to listen for incoming Bitbucket events. Events notifications are via DM in Mattermost. The Events don’t need separate configuration. 
+
+After your System Admin has [configured the Bitbucket plugin](#configuration), run `/bitbucket connect` in a Mattermost channel to connect your Mattermost and Bitbucket accounts.
+
+Once connected, you'll have access to the following features:
+
+* __Daily reminders__ - The first time you log in to Mattermost each day, get a post letting you know what issues and pull requests need your attention.
+* __Notifications__ - Get a direct message in Mattermost when someone mentions you, requests your review, comments on or modifies one of your pull requests/issues, or assigns you on Bitbucket.
+* __Post actions__ - Create a Bitbucket issue from a post or attach a post message to an issue. Hover over a post to reveal the post actions menu and click **More Actions (...)**.
+* __Sidebar buttons__ - Stay up-to-date with how many reviews, assignments, and open pull requests you have with buttons in the Mattermost sidebar.
+* __Slash commands__ - Interact with the Bitbucket plugin using the `/bitbucket` slash command. Read more about slash commands [here](#slash-commands).
+
+## Before You Start
+
+This guide assumes:
+
+- You have a Bitbucket account.
+- You're a Mattermost System Admin.
+- You're running Mattermost v5.25 or higher.
+
+## Configuration
+
+Configuration is started in Bitbucket and completed in Mattermost. 
+
+### Step 1: Register an OAuth Application in Bitbucket
+
+1. Go to https://bitbucket.org and log in.
+2. Visit the **Settings** page for your organization.
+3. Click the **OAuth** tab under **Access Management**.
+3. Click the **Add consumer** button and set the following values:
+   - **Name:** `Mattermost Bitbucket Plugin - <your company name>`.
+   - **Callback URL:** `https://your-mattermost-url.com/plugins/bitbucket/oauth/complete`, replacing `https://your-mattermost-url.com` with your Mattermost URL.
+   - **URL:** `https://github.com/kosgrz/mattermost-plugin-bitbucket`.
+4. Set:
+   - **Account:** `Email` and `Read` permissions.
+   - **Projects:** `Read` permission.
+   - **Repositories:** `Read` and `Write` permissions.
+   - **Pull requests:** `Read` permission.
+   - **Issues:** `Read` and `write` permissions.
+`Read` and `Write` permissions on Issues and Pull requests and `Read` on Repositories and Account for this OAuth consumer account. 
+5. Save.
+6. the **Key** and **Secret** in the resulting screen.
+7. Go to **System Console > Plugins > Bitbucket** and enter the **Bitbucket OAuth Client ID** and **Bitbucket OAuth Client Secret** you copied in a previous step.
+8. Hit **Save**.
```

Comment:
Add instructions to click `Regenerate` for Webhook Secret` and `At Rest Encryption Key` before saving